pkg/kubelet/kubelet_pods.go

```diff
@@ -1551,7 +1551,7 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 	}
 
 	// Make the latest container status comes first.
-	sort.Sort(sort.Reverse(kubecontainer.SortContainerStatusesByCreationTime(podStatus.ContainerStatuses)))
+	sort.Sort(sort.Reverse(kubecontainer.SortContainerStatusesByAttempt(podStatus.ContainerStatuses)))
```

Comment:
is that revers Less implementation was intended below, I wonder why it wasn't used here to eliminate the call to `Reverse`

Reply:
this can be simplified